examples/gnrc_networking/Makefile

```diff
@@ -29,6 +29,8 @@ USEMODULE += gnrc_icmpv6_echo
 USEMODULE += shell
 USEMODULE += shell_commands
 USEMODULE += ps
+USEMODULE += netstats_l2
+USEMODULE += netstats_ipv6
```

Comment:
I see. I wouldn't object, because `gnrc_networking` is IMO for showcasing our features, while `gnrc_minimal` is for showcasing our minamility.